airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/JobCancelButton.tsx

```diff
@@ -0,0 +1,53 @@
+import { faXmark } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import styled from "styled-components";
+
+import { LoadingButton } from "components";
+
+import useLoadingState from "hooks/useLoadingState";
+import { useCancelJob } from "services/job/JobService";
+
+import styles from "./JobCancelButton.module.css";
+
+const CancelButton = styled(LoadingButton)`
+  margin-right: 10px;
+  padding: 3px 7px;
+  z-index: 1;
+`;
+
+interface JobCancelButtonProps {
+  jobId?: number;
+}
+
+const JobCancelButton: React.FC<JobCancelButtonProps> = ({ children, jobId }) => {
+  const { isLoading, showFeedback, startAction } = useLoadingState();
+  const cancelJob = useCancelJob();
+
+  const onCancelJob = (event: React.SyntheticEvent) => {
+    event.stopPropagation();
+    return jobId ? startAction({ action: () => cancelJob(jobId) }) : null;
+  };
+
+  return (
+    <CancelButton
+      danger={!showFeedback}
```

Comment:
We should get rid of the use of showFeedback. the intent for this is to show a message next to the button or something when the event is complete, not to set a state of the button. Maybe we are missing this from the design and need to revisit, but the button currently looks broken when it changes to a different state for a brief second and then back to normal